### DISCUSSION TITLE
Hope to specify the default folder

### DISCUSSION THREAD

**Discussion by seyeeL** [coddingtonbear/obsidian-web#29] [Q&A] [answered]

**Comment by coddingtonbear** [comment, marked as answer]
This plugin interacts with Obsidian via [Obsidian Local REST API](https://coddingtonbear.github.io/obsidian-local-rest-api/)'s API, and you may have noticed that templates include an "API URL" field that _you_ have control over:

![image](https://user-images.githubusercontent.com/527661/185717981-8e69cc02-8e87-489f-a5a9-000d4cb904b3.png)

If you look at the docs for Obsidian Local REST API, you'd find that the path segment after `/vault/` is the filesystem path within your vault to which your file will be written.  In the case of the above example, it will create a file at your vault's root from your page's title.  If you wanted to place newly-created notes in a subfolder within your vault, just add that subfolder name after `/vault/` and before `{{filename page.title}}.md`.  For example, if you wanted to place your files within a subfolder named "captured pages", you could change the "API URL" field such that it reads instead `/vault/captured pages/{{filename page.title}}.md`.
